src/platform/intel/cavs/platform.c

```diff
@@ -336,16 +337,15 @@ static void platform_init_hw(void)
 	io_reg_write(DSP_INIT_IOPO,
 		IOPO_DMIC_FLAG | IOPO_I2S_FLAG);
 
-#ifndef __ZEPHYR__
 	io_reg_write(DSP_INIT_ALHO,
 		ALHO_ASO_FLAG | ALHO_CSO_FLAG);
 
 	io_reg_write(DSP_INIT_LPGPDMA(0),
 		LPGPDMA_CHOSEL_FLAG | LPGPDMA_CTLOSEL_FLAG);
 	io_reg_write(DSP_INIT_LPGPDMA(1),
 		LPGPDMA_CHOSEL_FLAG | LPGPDMA_CTLOSEL_FLAG);
-#endif
```

Comment:
We should not have any IP programming code in SOF now for Intel platforms. All IP programming must be done by Zephyr.